tests/python/pants_test/backend/python/tasks2/test_python_distribution_integration.py

```diff
@@ -0,0 +1,85 @@
+# coding=utf-8
+# Copyright 2017 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import (absolute_import, division, generators, nested_scopes, print_function,
+                        unicode_literals, with_statement)
+
+import os
+
+from pants.base.build_environment import get_buildroot
+from pants.util.process_handler import subprocess
+from pants_test.pants_run_integration_test import PantsRunIntegrationTest
+
+
+class PythonDistributionIntegrationTest(PantsRunIntegrationTest):
+  # The paths to both a project containing a simple C extension (to be packaged into a
+  # whl by setup.py) and an associated test to be consumed by the pants goals tested below.
+  superhello_project = 'examples/src/python/example/python_distribution/hello/superhello'
+  superhello_tests = 'examples/tests/python/example/python_distribution/hello/test_superhello'
+
+  def test_pants_binary(self):
```

Comment:
probably better to use the `skipif` decorator for these?

```
@unittest.skipIf(has_python_version(...), "no python ...")
def test_thing(...):
  pass
```

..but I would also expect python2.7 to be a strict requirement of pants' CI and test environments, so it's probably not even worth doing this check in the first place.

Reply:
I wrote it like this to leave room for extension when we want to test with Python 3 (similar to interpreter selection tests).

In this snippet, wouldn't we be doing `not has_python_version(...)` instead?

Reply:
✅